plugins/signalmonitor/signalhistorydelegate.cpp

```diff
@@ -49,7 +49,7 @@ SignalHistoryDelegate::SignalHistoryDelegate(QObject *parent)
     , m_totalInterval(0)
 {
     connect(m_updateTimer, SIGNAL(timeout()), this, SLOT(onUpdateTimeout()));
-    m_updateTimer->start(1000 / 25);
+    m_updateTimer->start(1000 / 15);
```

Comment:
you are not "optimizing the frame per seconds", you are "reducing the frames per second".